source/device/tensorrt/trt_executor.cc

```diff
@@ -34,30 +34,90 @@ EXPORT_FINISH
 
 #include <NvInferRuntime.h>
 
+#include <limits>
+#include <fstream>
+
 
 static Logger gLogger(nvinfer1::ILogger::Severity::kERROR);
 
 
+
+void SaveEngine(const std::string& fileName, nvinfer1::ICudaEngine* mEngine)
+{
+    if (fileName == "")
```

Comment:
```suggestion
    if (fileName.empty())
```
std:::string directly provides the method to checking.